docs/creature_template.md

```diff
@@ -577,7 +557,21 @@ Used to modify the base Level/Class armor of a creature.
 
 #### DamageModifier
 
-TODO!
+Used to modify the Minimum/Maximum damage of a creature.
+
+The formulas to calculate the damage output are:
+
+MINDAMAGE = ((([damage_base](https://www.azerothcore.org/wiki/creature_classlevelstats#damage_base) + ([attackpower](https://www.azerothcore.org/wiki/creature_classlevelstats#attackpower) / 14) * [BaseVariance](https://www.azerothcore.org/wiki/creature_template#BaseVariance)) * DamageModifier) * ([BaseAttackTime](https://www.azerothcore.org/wiki/creature_template#baseattacktime) / 1000))  
+MAXDAMAGE = (((([damage_base](https://www.azerothcore.org/wiki/creature_classlevelstats#damage_base) * 1.5) + ([attackpower](https://www.azerothcore.org/wiki/creature_classlevelstats#attackpower) / 14) * [BaseVariance](https://www.azerothcore.org/wiki/creature_template#BaseVariance)) * DamageModifier) * ([BaseAttackTime](https://www.azerothcore.org/wiki/creature_template#baseattacktime) / 1000))
+
+damage_base comes from the creature_classless table and takes its value either from [damage_base](https://www.azerothcore.org/wiki/creature_classlevelstats#damage_base), [damage_exp1](https://www.azerothcore.org/wiki/creature_classlevelstats#damage_exp1) or [damage_exp2](https://www.azerothcore.org/wiki/creature_classlevelstats#damage_exp2) according to the creature's value in [exp](http://www.azerothcore.org/wiki/creature_template#exp) (0 = base_damage, 1 = damage_exp1, 2 = damage_exp2).
+
+BaseAttackTime is either [BaseAttackTime](https://www.azerothcore.org/wiki/creature_template#baseattacktime) or [RangeAttackTime](https://www.azerothcore.org/wiki/creature_template#rangeattacktime) depending on the type of attack.
+
+attackpower is either [attackpower](https://www.azerothcore.org/wiki/creature_classlevelstats#attackpower) or [rangedattackpower](https://www.azerothcore.org/wiki/creature_classlevelstats#rangedattackpower) depending on the type of attack.
+
+BaseVariance is either [BaseVariance](https://www.azerothcore.org/wiki/creature_template#BaseVariance) or [RangeVariance](https://www.azerothcore.org/wiki/creature_template#RangeVariance) depending on the type of attack.
```

Comment:
Use only the small links, so for example [BaseVariance](#basevariance)

Reply:
Alright will add those later.